### ISSUE TITLE
Incorrect F256 math coprocessor registers

### ISSUE THREAD

**Issue by noyen1973** [Trinity-11/FoenixIDE#47]
The math coprocessor registers for the F256 are not the same as on the C256FMX/U.
```
; Unsigned Multiplier In A (16Bits), In B (16Bits), Answer (32Bits)
UNSIGNED_MULT_A_LO  = $DE00
UNSIGNED_MULT_A_HI  = $DE01
UNSIGNED_MULT_B_LO  = $DE02
UNSIGNED_MULT_B_HI  = $DE03
UNSIGNED_MULT_AL_LO = $DE10
UNSIGNED_MULT_AL_HI = $DE11
UNSIGNED_MULT_AH_LO = $DE12
UNSIGNED_MULT_AH_HI = $DE13

; Unsigned Divide Denominator A (16Bits), Numerator B (16Bits),
; Quotient (16Bits), Remainder (16Bits)
UNSIGNED_DIV_DEM_LO = $DE04
UNSIGNED_DIV_DEM_HI = $DE05
UNSIGNED_DIV_NUM_LO = $DE06
UNSIGNED_DIV_NUM_HI = $DE07
UNSIGNED_DIV_QUO_LO = $DE14
UNSIGNED_DIV_QUO_HI = $DE15
UNSIGNED_DIV_REM_LO = $DE16
UNSIGNED_DIV_REM_HI = $DE17

UNSIGNED_ADD_AL_LO = $DE08
UNSIGNED_ADD_AL_HI = $DE09
UNSIGNED_ADD_AH_LO = $DE0A
UNSIGNED_ADD_AH_HI = $DE0B
UNSIGNED_ADD_BL_LO = $DE0C
UNSIGNED_ADD_BL_HI = $DE0D
UNSIGNED_ADD_BH_LO = $DE0E
UNSIGNED_ADD_BH_HI = $DE0F
UNSIGNED_SUM_AL_LO = $DE18
UNSIGNED_SUM_AL_HI = $DE19
UNSIGNED_SUM_AH_LO = $DE1A
UNSIGNED_SUM_AH_HI = $DE1B

```

**Comment by dtremblay** [comment]
Bug confirmed.

**Comment by dtremblay** [comment]
Fixed in release 0.7.0.0.
